frontend/src/index.tsx

```diff
@@ -1,5 +1,13 @@
 import * as React from 'react'
 import { render } from 'react-dom'
+import * as ReactGA from 'react-ga'
 import { App } from './components/App/App'
+import { getGoogleAnalyticsAppId } from './utils/env'
+
+// Set up Google Analytics.
+const googleAnalyticsAppId = getGoogleAnalyticsAppId()
+console.log('Initializing Google Analytics with app ID: ' + googleAnalyticsAppId)
```

Comment:
is this supposed to be public?

Reply:
I think it's public whether or not we log it. Do you think we should remove the log output?